packages/open-payments/README.md

```diff
@@ -0,0 +1,197 @@
+| `privateKey`        | The the private EdDSA-Ed25519 key bound to the payment pointer, and used to sign the authenticated requests with. As mentioned above, a public JWK document signed with this key MUST be available at the `{paymentPointerUrl}/jwks.json` url.                                                                                                                                              |
+| `keyId`             | The key identifier of the given private key and the corresponding public JWK document.                                                                                                                                                                                                                                                                                                      |
+
+> **Note**
+>
+> To simplify EdDSA-Ed25519 key provisioning and JWK generation, you can use the `parseOrProvisionKey` and `generateJwk` methods from the [`http-signature-utils`](../http-signature-utils/README.md#usage) package.
+
+## Example
+
+As mentioned previously, Open Payments APIs can facilitate a payment between two parties.
+
+For example, say Alice wants to purchase a $50 product from a merchant called Shoe Shop on the Amazon Marketplace. If both parties have Open Payments enabled wallets, where Alice's payment pointer is `https://cloud-nine-wallet/alice`, and Shoe Shop's is `https://happy-life-bank/shoe-shop`, requests during checkout from Amazon's backend would look like this using the client:
+
+1. Create an Open Payments client
+
+In this case, since Amazon Marketplace wants to make requests that require authorization, it will need to create an `AuthenticatedClient`:
+
+```ts
+import { createAuthenticatedClient } from '@interledger/open-payments'
+
+const client = await createAuthenticatedClient({
+  paymentPointerUrl: 'https://amazon.com/usa',
+  keyId: KEY_ID,
+  privateKey: PRIVATE_KEY
+  // The public JWK with this key (and keyId) would be available at https://amazon.com/usa/jwks.json
+})
+```
+
+2.  Get `PaymentPointers`
+
+Grab the payment pointers of the parties:
+
+```ts
+const shoeShopPaymentPointer = await client.paymentPointer.get({
+  url: 'https://happy-life-bank/shoe-shop'
+})
+
+const customerPaymentPointer = await client.paymentPointer.get({
+  url: 'https://cloud-nine-wallet/alice'
+})
+```
+
+3. Create `IncomingPayment`
+
+Amazon's backend gets a grant to create an `IncomingPayment` on the merchant's wallet:
+
+```ts
+const incomingPaymentGrant = await client.grant.request(
+  { url: shoeShopPaymentPointer.authServer },
+  {
+    access_token: {
+      access: [
+        {
+          type: 'incoming-payment',
+          actions: ['read-all', 'create']
+        }
+      ]
+    }
+  }
+)
+
```

Comment:
I'd close the `ts` code block here, write somthing like 
"and creates an incoming payment using the access token from the grant"
and then I'd start the new `ts` code block with the incoming payment request.